spec/requests/evalutations_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+describe EvaluationsController do
+  describe '#current' do
+    let(:group) { create(:group) }
+
+    context 'when the user is not the group leader' do
+      it 'returns forbidden' do
+        user = create(:user)
+        login_as(user)
+
+        get "/groups/#{group.id}/evaluations/current"
+
+        expect(response).to have_http_status :forbidden
+      end
+    end
+
+    context 'when the user is the group leader' do
+      let(:user)    { group.leader.user }
+      before(:each) { login_as(user) }
+
+      it 'creates new evaluation' do
+        get "/groups/#{group.id}/evaluations/current"
+
+        new_evaluation = Evaluation.find_by(
+          group_id: group.id, semester: SystemAttribute.semester.to_s
```

Comment:
A little side note, it's not necessary to use `id` in this case
```suggestion
          group: group, semester: SystemAttribute.semester.to_s
```